src/lib/Scenes/Artwork/Components/__tests__/ArtworkTombstone-tests.tsx

```diff
@@ -35,6 +35,21 @@ describe("ArtworkTombstone", () => {
     expect(SwitchBoard.presentNavigationViewController).toHaveBeenCalledWith(expect.anything(), "/artist/andy-warhol")
   })
 
+  it("redirects to attribution class faq page when attribution class is clicked", () => {
+    const component = mount(
+      <Theme>
+        <ArtworkTombstone artwork={artworkTombstoneArtwork as any} />
```

Comment:
I'm not sure what error was being thrown, but is it possible to type this? (i.e. what happens if you get rid of the `as any`)